ab2d-properties-client/src/test/java/gov/cms/ab2d/properties/client/ProperyServiceMockTest.java

```diff
@@ -104,4 +114,38 @@ void testErrors() {
 
         wireMockServer.stop();
     }
+
+    @Test
+    void testDeleteCases() {
+        int port = 8066;
+        PropertiesClientImpl impl = new PropertiesClientImpl("http://localhost:" + port);
+        WireMockServer wireMockServer = new WireMockServer(port);
+        wireMockServer.start();
+
+        configureFor("localhost", port);
+
+        // test running delete with false
+        stubFor(delete(urlEqualTo("/properties/one")).willReturn(aResponse().withBody("false")));
+        assertThrows(PropertyNotFoundException.class, () -> impl.deleteProperty("one"));
+
+        // test running delete with true
+        stubFor(delete(urlEqualTo("/properties/one")).willReturn(aResponse().withBody("true")));
+        impl.deleteProperty("one");
```

Comment:
Do we need to assert something here? I could be incorrect, but I don't know if line 133 actually 'tests' anything otherwise.

For example:

`assertDoesNotThrow(PropertyNotFoundException.class, () -> impl.deleteProperty("one));`

Reply:
Ah, I thought I would get the "asser does not throw" behavior by default! I'll add that

Reply:
Added => https://github.com/CMSgov/AB2D-Libs/pull/386/commits/ed03614603ec8021aaa3637c61eef3bf1f072641